pkg/routing/localrouter.go

```diff
@@ -82,6 +83,7 @@ func (r *LocalRouter) ListNodes() ([]*livekit.Node, error) {
 }
 
 func (r *LocalRouter) StartParticipantSignal(ctx context.Context, roomName livekit.RoomName, pi ParticipantInit) (connectionID livekit.ConnectionID, reqSink MessageSink, resSource MessageSource, err error) {
+	prometheus.IncrementParticipantJoin(1, "rtc_init")
```

Comment:
Can we just make special versions of this function? Looks like `ParticipantJoin` is getting a few different flavours as the second argument. Wondering if it will make things easier to read/work with if the functions are separate.